mobile/lib/common/custom_app_bar.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class CustomAppBar extends StatelessWidget {
```

Comment:
Nice 👍 

Thanks for creating that component, we should use it everywhere 🙂

Can we rename it to `TenTenOneAppBar` instead of `CustomAppBar` ? 🙃

Reply:
ca87c51a183b6d4e22b2e9814cc247c99c7dc149